### PR TITLE
Enhance the skybox effect

### PR DESCRIPTION
**Pull request by whucj**
After this enhancement, the camera is always in the center of the skybox and will not be moved outside the skybox.

### CODE REVIEW

File: src/viewer/scene/mesh/Mesh.js [hunk 1]
```diff
@@ -212,6 +212,7 @@ class Mesh extends Component {
      * @param {Boolean} [cfg.edges=false] Indicates if the Mesh's edges are initially emphasized.
      * @param {Number[]} [cfg.colorize=[1.0,1.0,1.0]] Mesh's initial RGB colorize color, multiplies by the rendered fragment colors.
      * @param {Number} [cfg.opacity=1.0] Mesh's initial opacity factor, multiplies by the rendered fragment alpha.
+     * @param {Boolean} [cfg.background=false] Indicates if the Mesh is the background, such as skybox.
```
Comment (egghunters) [inline src/viewer/scene/mesh/Mesh.js:215]: Move two lines upper

Comment (alias0018) [inline src/viewer/scene/mesh/Mesh.js:215]: Move two lines upper.

File: src/viewer/scene/mesh/draw/DrawShaderSource.js [hunk 2]
```diff
@@ -454,6 +455,8 @@ function buildVertexDraw(mesh) {
     src.push("mat4 modelMatrix2          = modelMatrix;");
     if (stationary) {
         src.push("viewMatrix2[3][0] = viewMatrix2[3][1] = viewMatrix2[3][2] = 0.0;")
+    }else if(background){
```
Comment (alias0018) [inline src/viewer/scene/mesh/draw/DrawShaderSource.js:458]: style

File: src/viewer/scene/mesh/draw/DrawShaderSource.js [hunk 3]
```diff
@@ -454,6 +455,8 @@ function buildVertexDraw(mesh) {
...
+        src.push("viewMatrix2[3] = vec4(0.0,0.0,0.0,1.0);");
```
Comment (alias0018) [inline src/viewer/scene/mesh/draw/DrawShaderSource.js:459]: style

File: src/viewer/scene/mesh/Mesh.js (hunk 1 above)
Comment (alias0018) [reply src/viewer/scene/mesh/Mesh.js:215]: Indicates if the Mesh should act as background, e.g., it can be used for a skybox.

**Pull request merged**